static/app/views/dashboards/manage/utils.tsx

```diff
@@ -3,6 +3,11 @@ import localStorage from 'sentry/utils/localStorage';
 // TODO(nikkikapadia): delete this file and move contents to index.tsx
 export const SHOW_TEMPLATES_KEY = 'dashboards-show-templates';
 
+export const WIDGET_WIDTH = 300;
+export const PADDING = 16;
```

Comment:
```suggestion
export const PADDING = space(2);
```
or whichever spacing is correct!